policy/release_manager.md

```diff
@@ -0,0 +1,49 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming release.
```

Comment:
Maybe we should add info on a few edge cases or common cases on how this appointing works? Eg, you can't be made RM if you don't want it yourself. And what happens in the hypothetical situation that someone in the team objects an RM, or if there are more volunteers for the position than roles?

Reply:
I would be reluctant to include it, but would not object if there happens to be an agreeable statement. I think this document has a lot of explicit rules already, in comparison to what we had before. I personally made bad experiences trying to upfront-regulate a process that has never been tried before. So in some sense, my proposal is a declarative description of something that should match was we had in the past too, but turns it into an explicit role that a person can know to have, and adds the concept of having two people for resilience at all times.

I would prefer to stay vague on the appointment process. I consider it unlikely that we what try to make a person an RM that does not want to be in that role. If there is a competition for that role, we a 1) very lucky, and 2) I am confident that we can find a solution -- we are still very few people involved.

Once this has proven to work, we could formalize it more.